tests/ClientTest.php

```diff
@@ -770,6 +770,45 @@ public function testBuildWithStacktrace(): void
         $client->captureEvent(Event::createEvent());
     }
 
+    public function testBuildWithCustomStacktrace(): void
+    {
+        $options = new Options();
+
+        /** @var TransportInterface&MockObject $transport */
+        $transport = $this->createMock(TransportInterface::class);
+        $transport->expects($this->once())
+            ->method('send')
+            ->with($this->callback(function (Event $event): bool {
+                $stacktrace = $event->getStacktrace();
+
+                $this->assertInstanceOf(Stacktrace::class, $stacktrace);
+
+                /** @var Frame $lastFrame */
+                $lastFrame = array_reverse($stacktrace->getFrames())[0];
+
+                $this->assertSame(
+                    'MyApp.php',
+                    basename($lastFrame->getFile())
```

Comment:
Do we really need a call to `basename()` here? Isn't this value exactly the one you pass to the `buildFromBacktrace()` method?

Reply:
probably not, I copy pasted this code